Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/CriticalCmdletsUsageDetection.yaml

```diff
@@ -0,0 +1,57 @@
+id: 5170c3c4-b8c9-485c-910d-a21d965ee181
+name: VIP Mailbox manipulation
+description: |
+  'Alert if an high important Cmdlet is executed on a VIP Mailbox as those Cmdlets can be used for data exfiltration or mailbox access.'
+requiredDataConnectors:
+  - connectorId: ESI-ExchangeAdminAuditLogEvents
+    dataTypes:
+      - Event
+severity: Low
+queryFrequency: 30m
+queryPeriod: 1h
```

Comment:
Either the frequency should be 1h or query period should be 30m.

Reply:
Is it not allowed to cover a greater query period than the frequency ?
What about logs that are ingested to late by Log Analytics ? We saw in the past that few events was ingested with a delay, but at a correct TimeGenerated. Those events are not collected if I change the query period to 30 Minutes.

What's the BP ?

Reply:
In that case, we would need to manage the same in the query.
In the current situation, you will fire alert twice for the same condition.

In your query, you should have a filter of type | where ingestion_time() > ago(30m)